bin/varnishd/http2/cache_http2_deliver.c

```diff
@@ -92,6 +92,44 @@ h2_bytes(struct req *req, enum vdp_action act, void **priv,
 	return (0);
 }
 
+static struct h2_simple_response_msg {
+	const char	*p;
+	size_t		l;
+	unsigned	s;
+	const char	*r;
+} h2_simple_response_msg[] = {
+	[R_400] = { "\x92", 1, 400, "Bad Request"},
+#define SRESP(s, r) [R_##s] = { "\x18\x03" #s, 5, s, r}
+	SRESP(100, "Continue"),
+	SRESP(417, "Expectation Failed")
+};
```

Comment:
Nitpicking addressed (I would rebase before merge).
Other than that I don't know what you mean, sorry. Could you illustrate with code?